{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -1,20 +1,24 @@
-{%- if cookiecutter.test_matrix_configurator == "yes" -%}
-; this is just a bootstrap tox configuration, run tox or ./ci/bootstrap.py to have the full tox.ini
-
-[tox]
-envlist = bootstrap
-
 [testenv:bootstrap]
-basepython =
-    python
 deps =
     jinja2
     matrix
+    tox
 skip_install = true
 commands =
     python ci/bootstrap.py
+{% if cookiecutter.test_matrix_configurator == "yes" %}
+basepython =
+    python
 passenv =
     *
+{% endif %}
```

Comment:
I think the endif should be before the passenv.

Reply:
Fair enough. I didn't have any particular reason for passenv to be specific to if cookiecutter.test_matrix_configurator == "yes", I just put it there because previously it *was* specific to if cookiecutter.test_matrix_configurator == "yes" and I figured maybe there was a reason.